app/models/event.rb

```diff
@@ -58,5 +58,8 @@ def metadata
   end
 
   json do
+    translate(
+      occurred_at: :occured_at
+    )
```

Comment:
I'm not sure if fixing the column name is worth the pain, but at least we can let messages be sent with the correct spelling.

Reply:
😆  interacting with this field has genuinely made me forget how to spell it